### PR TITLE
Replace circle.yml skeleton file

### PR DESCRIPTION
**Pull request by rafkhan**

### CODE REVIEW

File: src/api/0.1/project-fs/skeletons/circle.yml [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 dependencies:
-  pre:
-    - npm install aws-sdk
-  post:
-    - ./$CLUSTERNATOR_DIR/decrypt.sh
-
+    pre:
+      - 'sudo apt-get install gnupg'
+      - 'npm install aws-sdk'
+      - 'npm install -g clusternator the-clusternator-runtime-aws'
```
Comment (SethDavenport) [inline src/api/0.1/project-fs/skeletons/circle.yml]: OK for now but eventually we'd like `clusternator init` to do an `npm install --save-dev clusternator-runtime-aws` instead.

File: src/api/0.1/project-fs/skeletons/circle.yml [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+        - docker
 dependencies:
-  pre:
-    - npm install aws-sdk
-  post:
-    - ./$CLUSTERNATOR_DIR/decrypt.sh
-
+    pre:
+      - 'sudo apt-get install gnupg'
+      - 'npm install aws-sdk'
```
Comment (SethDavenport) [inline src/api/0.1/project-fs/skeletons/circle.yml]: This should be a dependency of the runtime itself?

File: src/api/0.1/project-fs/skeletons/circle.yml [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-      - 'node ./.clusternator/docker-build.js pr'
-  deploy:
-    branch: master
-    owner: $OWNER
-    commands:
-      - 'node ./.clusternator/docker-build.js deploy master'
+    pr:
+        branch: '/^(?!master$).*/'
+        commands:
+          - 'clusternator-runtime-aws ./ $CLUSTERNATOR_HOST pr'
```
Comment (bennett000) [inline src/api/0.1/project-fs/skeletons/circle.yml]: does `$CLUSTERNATOR_HOST` have to be a circle environment variable?

**Comment by bennett000** [comment]
Nice, I'm glad you were able to modify the npm shell bit, that's great, :shipit:

**Pull request merged**